src/components/section/LogInCard.tsx

```diff
@@ -21,50 +21,58 @@ export const LogInCard: React.FC = () => {
-                      event.timeZone
-                    )
-                  : "Loading..."}
+          {event ? (
+            <div className="flex w-full flex-col gap-2 border-b-[1px] border-gray-300 p-5">
+              <div className="flex flex-row items-center gap-1 text-sm text-gray-500">
+                <MdOutlineCalendarToday />
+                <div>
+                  {event
+                    ? formatOccurring(
+                        occurringDaysArray ?? [],
+                        event.type === "DAYSOFWEEK",
+                        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                        event.timeZone
+                      )
+                    : "Loading..."}
+                </div>
+                <MdOutlineAccessTime className="ml-1" />
+                <div>
+                  {event
+                    ? // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+                      formatTimespan(event.begins, event.ends, event.timeZone)
+                    : "Loading..."}
+                </div>
+                <HiOutlineGlobe className="ml-1" />
+                <div>
+                  {/* eslint-disable-next-line @typescript-eslint/no-unsafe-argument */}
+                  {event ? formatTimeZoneTag(event.timeZone) : "Loading..."}
+                </div>
               </div>
-              <MdOutlineAccessTime className="ml-1" />
-              <div>
-                {event
-                  ? // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-                    formatTimespan(event.begins, event.ends, event.timeZone)
-                  : "Loading..."}
+              <div className="text-3xl font-semibold text-black">
+                {event?.name ?? "Loading..."}
               </div>
-              <HiOutlineGlobe className="ml-1" />
-              <div>
-                {/* eslint-disable-next-line @typescript-eslint/no-unsafe-argument */}
-                {event ? formatTimeZoneTag(event.timeZone) : "Loading..."}
+              <div className="text-sm font-normal text-black">
+                {event &&
+                  (event._count.participant === 1 ? (
+                    <div>No one except host has filled yet</div>
+                  ) : (
+                    <div>
+                      {event._count.participant} people filled including host
+                    </div>
+                  ))}
+                {!event && "Loading..."}
               </div>
             </div>
-            <div className="text-3xl font-semibold text-black">
-              {event?.name ?? "Loading..."}
+          ) : (
+            <div className="flex flex-1 flex-col gap-2 border-b-[1px] border-gray-300 p-5">
+              <div className="skeleton h-5 max-w-[400px]" />
+              <div className="skeleton h-9 max-w-[600px]" />
+              <div className="skeleton h-5 max-w-[300px]" />
             </div>
```

Comment:
A left to right running highlight on the gray skeleton would be much better imo, but this looks fine as well!

Reply:
Making such animation requires more than just a Tailwind CSS class as far as I know. We can leave this to a later update.